templates/debian9/Dockerfile.template

```diff
@@ -193,7 +193,8 @@ COPY --from=plugins /plugins/* /usr/lib/rabbitmq/plugins/
 COPY --from=plugins /licenses/* /usr/share/doc/rabbitmq-plugins/
 
 COPY docker-entrypoint.sh /usr/local/bin/
-RUN ln -s usr/local/bin/docker-entrypoint.sh / # backwards compat
+RUN ln -s usr/local/bin/docker-entrypoint.sh / \
+	&& chmod +x /usr/local/bin/docker-entrypoint.sh # backwards compat
```

Comment:
Maybe its better to add owner when copying and do not use chmod
`COPY --chown=rabbitmq docker-entrypoint.sh /usr/local/bin/ ... `

Reply:
As accorded in person. I will remove this line and grant the execution permission on local file.